Proofs/SHA512/Sha512_block_armv8.lean

```diff
@@ -80,6 +81,20 @@ theorem sha512_program_test_2_sym (s0 s_final : ArmState)
 
 -- Test 3:
 
+theorem Aligned_BitVecAdd_64_4 {x : BitVec 64} {y : BitVec 64}
+  (x_aligned : Aligned x 4)
+  (y_aligned : Aligned y 4)
+  :
+  Aligned (x + y) 4 := by
+  simp_all [Aligned]
+  bv_decide
+
+@[state_simp_rules]
+theorem CheckSPAlignment_BitVecAdd_64_4 {st : ArmState} {y : BitVec 64}
+  (x_aligned : CheckSPAlignment st)
+  (y_aligned : Aligned y 4)
+  :
+  Aligned ((r (StateField.GPR 31#5) st) + y) 4 := Aligned_BitVecAdd_64_4 x_aligned y_aligned
```

Comment:
@nwetzler Let's move these into `Proofs/Insts/Common.lean` so that they're available everywhere. But that file doesn't import LeanSAT, so perhaps `bv_decide` can be replaced by `bv_omega` or something, if it is easy enough to do?

Reply:
Yeah, that was the downside of moving them.  I'll try with `bv_omega`.